lib/Raven/ReprSerializer.php

```diff
@@ -29,7 +29,11 @@ protected function serializeValue($value)
         } elseif (is_integer($value) || is_float($value)) {
             return (string) $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
-            return 'Object '.get_class($value);
+            $repr = 'Object ' . get_class($value);
+            if (method_exists($value, 'toSentry')) {
+                $repr .= ' ' . $value->toSentry();
```

Comment:
Yeah it would be great to have some sort of type indication for variables, but looking at the [docs](https://docs.sentry.io/clientdev/interfaces/stacktrace/) that currently is not possible.

Would maybe then leave it to the user to be specific about the type of the variable in their return from `toSentry`.

How is this done in Python @dcramer? Maybe the type indication for variables could be a nice addition to the Sentry server/protocol.